fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -265,37 +273,37 @@ private void extractResults(SqlRowSet rowSet, String[] resultColumnNames, List<J
     }
 
     @NotNull
-    private SqlRowSet callFilteredMysql(String sql, String valueFilter, String filterColumnType) {
+    private SqlRowSet callFilteredMysql(String sql, Object valueFilter, String filterColumnType) {
```

Comment:
Please use specific type!

Reply:
Same old validateColumn method, that I have to refactor then :cry: